footprints/main/tests/test_viaf.py

```diff
@@ -0,0 +1,69 @@
+from django.test.testcases import TestCase
```

Comment:
This doesn't use the database at all so using `unittest.TestCase` instead of `django.test.TestCase` would avoid creating and rolling back a database transaction before/after each test.